pkg/controller/spire-oidc-discovery-provider/configmaps.go

```
@@ -46,8 +51,8 @@ func GenerateOIDCConfigMapFromCR(cr *v1alpha1.SpireOIDCDiscoveryProvider) (*core
 		"log_level": "debug",
 		"serving_cert_file": map[string]string{
 			"addr":           ":8443",
-			"cert_file_path": "/certs/tls.crt",
-			"key_file_path":  "/certs/tls.key",
+			"cert_file_path": "/etc/oidc/tls/tls.crt",
+			"key_file_path":  "/etc/oidc/tls/tls.key",
 		},
 		"workload_api": map[string]string{
 			"socket_path":  "/spiffe-workload-api/" + agentSocketName,
```

Prefer using named constants instead of hardcoded values in the code to improve readability, maintainability, and reduce the risk of errors.